src/lib.rs

```diff
@@ -525,6 +525,8 @@ pub struct Runner<'a, PWR, SPI> {
     ioctl_id: u16,
     sdpcm_seq: u8,
     backplane_window: u32,
+
+    tx_seq_max: u8,
```

Comment:
perhaps name this `sdpcm_seq_max` for consistency with `sdpcm_seq`? Or `sdpcm_tq_seq, sdpcm_tx_seq_max` to make it clear they're about `tx`.